content/docs/blog/posts/2025-06-16-release-v0.16.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Shipwright v0.16 Is Here"
+date: 2025-06-16T09:41:00Z
+draft: false
+author: "Enrique Encalada ([@qu1queee](https://github.com/qu1queee))"
+---
+
+We are happy to announce the latest release of Shipwright's main projects - `v0.16.0`.
+In this release, besides new features and fixes, we're excited to announce that we've launched
+a brand new [website](https://shipwright.io/)! It's been redesigned from the ground up to better showcase the project,
+make it easier to navigate, and help you find everything you need. Special thank you to [@rxinui](https://github.com/rxinui) for this core contribution!
+
+## Key Features
+
+Below are the key features in this release:
+
+### Improving Git Clone
+
+The `source.git.depth` was added to the `Build` resource to specify the depth of the Git history. 
+If not specified the default value is 1 which means that no history is cloned at all, being the 
+fastest way to clone a Git repository. Any value greater than 1 will create a clone with the specified depth.
+For a full git history clone, depth must be set to 0.
+
+### Optimizing Controller Memory Footprint
+
+Improve the Shipwright controller’s memory efficiency by restricting the manager cache to only TaskRuns and Pods associated with the controller. This is achieved by leveraging the `buildrun.shipwright.io/name` label to identify relevant resources. By narrowing the scope of what the manager caches, we significantly reduce memory usage, resulting in a dramatic improvement in overall controller performance.
+
+### CLI: Maintenance Update
+
+The CLI was updated to support Build v0.16.0 APIs.
```

Comment:
Actually, the CLI under the cover now uses the Beta APIs of shipwright. This should make things slightly faster as the conversion webhook is out of the picture when using the CLI. https://github.com/shipwright-io/cli/pull/304